pkg/landscape/app/notify.hoon

```diff
@@ -22,21 +22,29 @@
   $:  =provider-state
       =client-state
   ==
++$  base-state-1
+  $:  notifications=(map uid notification)
+      base-state-0
+  ==
 ::
 +$  state-0
   [%0 base-state-0]
 ::
 +$  state-1
   [%1 base-state-0]
 ::
++$  state-2
+  [%2 base-state-1]
+::
```

Comment:
I think it's more common to just bump directly to `base-state-2`, as in "the `base-state` that was introduced during state version `%2`. Seems better than getting versioning for parts of the state completely out of sync.